__tests__/app.test.js

```diff
@@ -105,4 +105,55 @@ describe('/api/articles',()=>{
       })
     })
   })
+})
+describe("/api/articles/:article_id/comments",()=>{
+  test("GET:200 should return all comments for an article",()=>{
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then((response)=>{
+      const body = response.body
+      expect(body.comments.length).toBe(11)
+      body.comments.forEach((comment)=>{
```

Comment:
The test for this endpoint is great, checking the array length matches what we expect before we start looping and checking the properties stops us from getting any false positives if the array is empty. 😄